lib/import/import_samples.rb

```diff
@@ -177,6 +177,15 @@ def molecule_not_exist(molecule)
     molecule.nil?
   end
 
+  # format row[field] for melting and boiling point
+  def process_row(row_field)
+    # Regex checks for a range of numbers that are separated by a dash, or a single number
+    matches = row_field.scan(/^(-?\d+(?:[.,]\d+)?)(?:\s*-\s*(-?\d+(?:[.,]\d+)?))?$/).flatten.compact
```

Comment:
the regex is quite complicated to read but i have no idea how to make it more readable :)